docs/variables.md

```diff
@@ -2074,6 +2169,22 @@ Specifies whether MyRocks should re-read the data file
 as soon as it is created to verify correctness.
 Enabled by default.
 
+### `rocksdb_partial_index_ignore_killed`
+
+| Option       | Description               |
+|--------------|---------------------------|
+| Command-line | --rocksdb-partial-index-ignore-killed |
+| Dynamic      | Yes                       |
+| Scope        | Global                    |
+| Data type    | Boolean                   |
+| Default      | ON                        |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+The dafault value is `ON` which means this variable is enabled.
```

Comment:
Should be definition of the variable and then the default value.

Reply:
fixed